application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -99,9 +101,13 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
             .setActionRow(Button.of(ButtonStyle.DANGER, generateComponentId(userID), "Delete"))
             .queue();
     }
-
+    /**
+     * Converts inline latex like:
+     * 'hello $\frac{x}{2}$ world' to full latex '\text{hello }\frac{x}{2}\text{ world}'.
```

Comment:
Btw, you can do `{@code ...}` in javadoc. Might make the two examples more readable if u wrap it around them.

Reply:
I meant it more like
```java
Converts inline latex like:
{@code hello $\frac{x}{2}$ world}
to full latex
{@code \text{hello}\frac{x}{2}\text{ world}}.
```

Reply:
Now its correct. Could you please double check whether Intellij actually renders the Javadoc correctly on mousehover? Maybe we have to escape the `{...}` of `frac` and `text` for it to not freak out.

Just tell me whether it works, then I will close this.

Reply:
(shown in discord, looks good).